pymc3/sampling.py

```diff
@@ -265,20 +265,26 @@ def _choose_backend(trace, chain, shortcuts=None, **kwds):
     except KeyError:
         raise ValueError('Argument `trace` is invalid.')
 
+def _make_parallel(arg, njobs):
+    if not shape(arg):
```

Comment:
Feel free to propose an alternative. There should not be any pandas data structures being passed as seed or starting values, and I can't think of a situation where `np.shape` would return a misleading result (i.e. one that would not be evaluated to an empty tuple if there were no dimensionality to the data structure), in any case.